### PR TITLE
helm: Add certgen.generateCA value

### PR DESCRIPTION
**Pull request by sderoe**
Allow creation of secrets without CA creation.

### CODE REVIEW

File: install/kubernetes/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl [hunk 1]
```diff
@@ -19,7 +19,7 @@ spec:
             {{- if .Values.debug.enabled }}
             - "--debug"
             {{- end }}
-            - "--ca-generate"
+            - "--ca-generate={{ .Values.certgen.generateCA }}"
```
Comment (giorio94) [inline install/kubernetes/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl:22]: The same change shall be applied to `install/kubernetes/cilium/templates/hubble/tls-cronjob/_job-spec.tpl` as well.

**Review by giorio94 — CHANGES_REQUESTED** [review]
Thanks! The patch looks good to me. Could you please also extend it to the Hubble certificates generation (see comment inline)?

**Comment by sderoe** [comment]
@giorio94 Thank you. I have updated the hubble template as well

**Review by giorio94 — APPROVED** [review]
Thanks!

**Review by gandro — APPROVED** [review]

**Comment by gandro** [comment]
Not sure why the Helm smoke test is failing. I don't see any error message. Could you run `make -C install/kubernetes` locally and commit any differences if there are any?

**Comment by sderoe** [comment]
@gandro There was a diff so I have committed it. Helm check is waiting to run

**Comment by gandro** [comment]
Seems like there are still some issues. Checkpatch says that the Sign-Off line is inconsistent:

> Error: ERROR:NO_AUTHOR_SIGN_OFF: Missing Signed-off-by: line by nominal patch author 'john-r-swyftx <john.roche@swyftx.com.au>'

And it seems the documentation needs some regeneration as well:

> HINT: to fix this, run 'make -C Documentation update-helm-values'

**Comment by sderoe** [comment]
Just ran `make -C Documentation update-helm-values` as well

**Comment by sderoe** [comment]
Updated the email as well. Didn't realise it was tested v account email

**Pull request merged**